packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -534,6 +534,13 @@ export class ZWaveController extends TypedEventEmitter<ControllerEventCallbacks>
 		this.driver.cacheSet(cacheKeys.controller.supportsSoftReset, value);
 	}
 
+	public get isAnyFirmwareUpdateInProgress(): boolean {
```

Comment:
Should we call this `isOTAFirmwareUpdateInProgress` in order to distinguish node updates from the OTW update for the controller (which I'll eventually add)?

Also should this be a getter/property or a method? Feels more like a method at first glance.

Reply:
good idea, and we can make it a method